mlx_lm/utils.py

```diff
@@ -202,15 +201,27 @@ def class_predicate(p, m):
             model,
             group_size=quantization["group_size"],
             bits=quantization["bits"],
+            mode=quantization.get("mode", "affine"),
             class_predicate=class_predicate,
         )
+
+    if (quantization := config.get("quantization", None)) is not None:
+        _quantize(quantization)
+
     elif quantization_config := config.get("quantization_config", False):
         # Handle legacy quantization config
         quant_method = quantization_config["quant_method"]
         if quant_method == "bitnet":
             from .models.bitlinear_layers import bitnet_quantize
 
             model = bitnet_quantize(model, quantization_config)
+        elif quant_method == "mxfp4":
+            quantization = {"group_size": 32, "bits": 4, "mode": "mxfp4"}
```

Comment:
Would it be possible to set these values as default for convert as well? I am currently passing many arguments shown below. Simply passing `--q-mode mxfp4` didn't work.

```
-q --q-bits 4 --q-group-size 32 --q-mode mxfp4 --dtype bfloat16
```